include/deal.II/lac/constraint_matrix.templates.h

```diff
@@ -624,14 +624,26 @@ distribute_local_to_global (const Vector<LocalType>      &local_vector,
                             VectorType                   &global_vector,
                             const FullMatrix<LocalType>  &local_matrix) const
 {
+  distribute_local_to_global(local_vector,local_dof_indices,local_dof_indices, global_vector, local_matrix);
+}
+
```

Comment:
Please add another two empty lines. We usually have three empty lines between functions.